package.json

```diff
@@ -9,7 +9,9 @@
     "packages/*"
   ],
   "scripts": {
-    "lint": "eslint .",
+    "lint": "yarn lint:unlink; eslint .; yarn lint:link",
+    "lint:unlink": "rm packages/cli/src/emails",
```

Comment:
why are these changes necessary?

Reply:
The symlink causes the linter to misbehave.  Basically, the linter complains that each file in packages/cli/src/emails is not covered by a tsconfig file, but the path it lists is the resolved path of each file in generator_templates/ts/emails/.  There is no way to indicate to the linter to ignore the symlink because it first attempts to resolve the symlink, so you can either do this or you have to tell the linter to ignore the files that are in generator_templates/ts/emails/.  You can try removing this and running the linter to see the error messages